Competition2017/tests/competition/subsystem/shift/ShiftSubsystemTest.java

```diff
@@ -0,0 +1,27 @@
+package competition.subsystem.shift;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import competition.subsystems.shift.ShiftSubsystem.Gear;
+
+public class ShiftSubsystemTest extends ShiftTestBase {
+
+    @Before
+    public void setUp() {        
+        super.setUp();
+    }
+    
+    @Test
+    public void setHighGearTest() {
+        shiftSubsystem.setGear(Gear.HIGHGEAR);
+        verifyGear(Gear.HIGHGEAR);
+    }
+    
+    @Test
+    public void setLowGearTest() {
+        shiftSubsystem.setGear(Gear.LOWGEAR);
+        verifyGear(Gear.LOWGEAR);
+    }
+    
```

Comment:
You should have a least one test where you set Low->High->Low->High, so you test all transitions.